src/safeposix/syscalls/net_calls.rs

```diff
@@ -1859,6 +1859,40 @@ impl Cage {
         }
     }
 
+    /// ## ------------------SELECT SYSCALL------------------
+    /// ### Description
+    /// The `select_syscall()` allows a program to monitor multiple file descriptors, waiting until one or more of the file descriptors become "ready"
+    /// for some class of I/O operation (e.g., input possible).  A file descriptor is considered ready if it is possible to perform a
+    /// corresponding I/O operation (e.g., `read_syscall()`) without blocking.
+    
+    /// ### Function Arguments
+    /// The `select_syscall()` receives five arguments:
+    /// * `nfds` - This argument should be set to the highest-numbered file descriptor in any of the three sets, plus 1.  The
+    ///           indicated file descriptors in each set are checked, up to this limit.
```

Comment:
Adding the feedback that I also received for Rust Documentation. 
1. We need to wrap the comments to be <80 characters wide.
2. The maximum number of tabs can be 4, so we need to remove the extra whitespace here.

Reply:
@qianxichen233 Can you please reformat this code with `cargo fmt` command as looks like this section might need a bit of formatting.